atcodertools/common/language.py

```diff
@@ -82,10 +82,10 @@ def from_name(cls, name: str):
     name="cpp",
     display_name="C++",
     extension="cpp",
-    submission_lang_pattern=re.compile(".*C\\+\\+14 \\(GCC.*"),
+    submission_lang_pattern=re.compile(".*C\\+\\+ \\(GCC.*"),
```

Comment:
旧ジャッジでC++14が使えるようにバージョンで分けるのはどうでしょうか。
```suggestion
    submission_lang_pattern=re.compile(".*C\\+\\+ \\(GCC 9.*|.*C\\+\\+14 \\(GCC 5.*"),
```